docs/mobile/concepts-app-config.mdx

```diff
@@ -24,15 +24,15 @@ While [layouts](concepts-layout.mdx) depict the visual structure of how componen
 Every component within the viewer that requires functional, compositional content needs to be configured within the viewer's corresponding app config.
 
 :::tip Want to follow along?
-Check out the [Quick Start](quick-start.mdx) and [edit the default app config and layout](quick-start.mdx#editing-the-layout-and-app-config).
+Check out the [Quickstart](quick-start.mdx) and [edit the default app config and layout](quick-start.mdx#editing-the-layout-and-app-config).
 :::
 
 ## Anatomy of an App config.
 
 App config files are composed of two main components:
 
-1. High level configuration values
-2. Configuration for individual **app items**
+1. High level configuration values.
```

Comment:
I'm leaning towards excluding periods at the end of bullet/number lists. AFAIK they can be used for items that form complete sentences (AFAIK), but aren't necessary i don't believe.

Thoughts?

Reply:
There is a colon in the previous sentence, so AFAIK the sentence should be closed with a period somehow? Maybe adding a comma to the preceding list elements would make it less weird.

This might be a finicky english grammar/styling question we could get a docs team members advice on.

Reply:
Yeah we can get their opinion on that. We can possibly do that when do we the final scan and polish before we call MVP done.

Reply:
I went with the following: if the bullet/number list isn't a full sentence, don't include the period. If it is a full sentence/complete thought, then do include the period. I think that should be sufficient for now and we can iterate if we find that doesn't work.